### PR TITLE
Delete form-builders css references

### PR DESCRIPTION
**Pull request by tugrulcan**
Issue #1766
form-builder's v2.2 removes the separate css file see:
https://github.com/kevinchappell/formBuilder/blob/master/CHANGELOG.md

### CODE REVIEW

File: app/config/assetic.yml [hunk 1]
```diff
@@ -63,8 +63,6 @@ assetic:
         - '%kernel.root_dir%/../web/assets/app/css/_history.css'
         - '%kernel.root_dir%/../web/vendor/sweetalert/dist/sweetalert.css'
         - '%kernel.root_dir%/../web/vendor/fancybox/source/jquery.fancybox.css'
-        - 'https://formbuilder.online/assets/css/form-builder.min.css'
```
Comment (behram) [inline app/config/assetic.yml]: @tugrulcan you thought awesome ;) but, we using this style assets on project. can you replace with existing ones

**Review by behram — CHANGES_REQUESTED** [review]

File: app/config/assetic.yml (hunk 1 above)
Comment (tugrulcan) [inline app/config/assetic.yml]: I added the form-builder under  web/assests/app/css/formBuilder-2.1.2

**Comment by tugrulcan** [comment]
formbuilder added under web/assests/app/css/formBuilder-2.1.2

**Comment by tugrulcan** [comment]
@behram please review :)

**Comment by behram** [comment]
i am testing this pr now.

**Comment by behram** [comment]
@tugrulcan i used formBuilder bower package beside static assets. thanks for your work. issue resolved with another pr.

related pr;
 - https://github.com/academic/vipa/pull/1768

**Pull request closed without merging**